### DISCUSSION TITLE
v3.2.86

### DISCUSSION THREAD

**Discussion by danharrin** [filamentphp/filament#13100] [General]
## What's Changed
* Rename extraFieldWrapperAttributes to extraEntryWrapperAttributes by @jyrkidn in https://github.com/filamentphp/filament/pull/13034
* Update Albanian pagination translation by @dajanvulaj in https://github.com/filamentphp/filament/pull/13033
* Fix issue when Cluster large Submenu items makes content grid align items centered instead of start by @eelco2k in https://github.com/filamentphp/filament/pull/13052
* The import or export notification header depends on getCompletedNotificationTitle function by @Sergey-Karapetyan in https://github.com/filamentphp/filament/pull/13040
* Add documentation for publishing stubs by @dissto in https://github.com/filamentphp/filament/pull/13043
* Minor fixes on French translations by @abournavaud in https://github.com/filamentphp/filament/pull/13072
* Update CODEOWNERS by @saade in https://github.com/filamentphp/filament/pull/13076
* Auth classes rate limit notification method by @olehasoft in https://github.com/filamentphp/filament/pull/13056
* Fix importer column by @jlaperaltab in https://github.com/filamentphp/filament/pull/13077
* Add `Support & Help` page to documenation by @dissto in https://github.com/filamentphp/filament/pull/13078
* Fix custom size CSS class by @underdpt in https://github.com/filamentphp/filament/pull/13038
* fix: typo 'Complile' on 03-charts.md by @victornalves in https://github.com/filamentphp/filament/pull/13088
* fix: handle panel ID correctly in make:filament-resource command by @mwguerra in https://github.com/filamentphp/filament/pull/13082
* feat: set locale on DateTimePicker field by @darkclow4 in https://github.com/filamentphp/filament/pull/13099

## New Contributors
* @dajanvulaj made their first contribution in https://github.com/filamentphp/filament/pull/13033
* @eelco2k made their first contribution in https://github.com/filamentphp/filament/pull/13052
* @Sergey-Karapetyan made their first contribution in https://github.com/filamentphp/filament/pull/13040
* @jlaperaltab made their first contribution in https://github.com/filamentphp/filament/pull/13077
* @victornalves made their first contribution in https://github.com/filamentphp/filament/pull/13088
* @mwguerra made their first contribution in https://github.com/filamentphp/filament/pull/13082
* @darkclow4 made their first contribution in https://github.com/filamentphp/filament/pull/13099

**Full Changelog**: https://github.com/filamentphp/filament/compare/v3.2.85...v3.2.86

<hr /><em>This discussion was created from the release <a href='https://github.com/filamentphp/filament/releases/tag/v3.2.86'>v3.2.86</a>.</em>

**Comment by fatihdurmaz** [comment]
rich editor v2 have a bug. not resetting after create another in modal.
